integration-test-groups/foundation/controlbus/pom.xml

```diff
@@ -58,6 +62,20 @@
     </dependencies>
 
     <profiles>
+        <!--
+        <profile>
+            <id>jvm</id>
```

Comment:
Do we need this profile given the JVM specific tests we're moved elsewhere?

Reply:
Forgot I'd left that in - removed.